mrp_multi_level/security/mrp_multi_level_security.xml

```diff
@@ -7,6 +7,16 @@
         <field name="category_id" ref="base.module_category_hidden"/>
     </record>
 
+    <record id="group_mrp_multi_level_run" model="res.groups">
+        <field name="name">Run MRP Manually</field>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]" />
+        <field name="category_id" ref="base.module_category_hidden" />
```

Comment:
Add admin user here by default, I will be the most common when you have a fresh installation.

Reply:
basically this: <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"/>

Reply:
:)  can I do the same thing for the group "Show full accounting Features"? :innocent:

Reply:
hahaha that would be the best fix ever ;)